content/doc/book/installing/_jenkins-command-parameters.adoc

```diff
@@ -139,27 +139,51 @@ the keystore to PEM] and use following command line options:
+The link:https://tools.ietf.org/html/rfc7540[HTTP/2 protocol] allows web servers to reduce latency over encrypted connections by pipelining requests, multiplexing requests, and allowing servers to push in some cases before receiving a client request for the data.
+The Jetty server used by Jenkins supports HTTP/2 with the addition of the Application-Layer Protocol Negotiation (ALPN) TLS extension.
+The ALPN TLS extension is connected to the specific Jetty version and has specific requirements depending on the Java version.
 
-You can download the required version from the link:https://repo.maven.apache.org/maven2/org/mortbay/jetty/alpn/alpn-boot/[Apache maven repository].
+==== Java 8u252 and Java 11
 
-Then you have to include it on jvm start:
+Java 8 update 252 (and later) and all versions of Java 11 can run the ALPN TLS extension by installing the Jetty ALPN java server jar and passing it as a java command line argument.
+Steps to install the extension are:
+
+* Identify the Jetty version included in your Jenkins server by reviewing the link:/changelog/[Jenkins changelog] for your release
+* Locate the Java version on the "System Information" page of "Manage Jenkins" to confirm it is Java 11 or 8u252 (or later)
+* Download the link:https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-alpn-java-server[jetty-alpn-java-server] with the version number matching the Jetty version bundled with your Jenkins version
+* Place the jetty-alpn-java-server.jar file in a directory accessible to the JVM
+* Add `--extraLibFolder=/path/to/extra/lib/folder` to the Java command line arguments that start Jenkins
+
+[source,bash]
+----
+java --extraLibFolder=/opt/java/jetty-alpn-java-server-9.4.27.v20200227.jar \
+    -jar target/jenkins.war \
+    --http2Port=9090
+----
+
+==== Java 8u242 and earlier
+
+Java 8 update 242 (and earlier) can run the ALPN TLS extension by installing the Jetty ALPN java server jar into the Java boot classpath.
+Steps to install the extension are:
+
+* Identify the Jetty version included in your Jenkins server by reviewing the link:/changelog/[Jenkins changelog] for your release
+* Locate the Java version on the "System Information" page of "Manage Jenkins" to confirm that it is 8u242 or earlier
+* Download the link:https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-alpn-java-server[jetty-alpn-java-server] with the version number matching the Jetty version bundled with your Jenkins version
+* Place the jetty-alpn-java-server.jar file in a directory accessible to the JVM
+* Add jetty-alpn-java-server.jar to the JVM boot classpath by adding `-Xbootclasspath/p:/path/to/jetty-alpn-java-server.jar` to the Java command line arguments that start Jenkins
 
 [source,bash]
 ----
-java -Xbootclasspath/p:alpn-boot-8.1.12.v20180117.jar \
+java -Xbootclasspath/p:/opt/java/jetty-alpn-java-server-9.4.27.v20200227.jar \
```

Comment:
for 8u242 and earlier must be still alpn-boot

Reply:
Corrected in c5cc23f09c819d16f3ce451596d7cd6c6aa3da47